client/js/lounge.js

```diff
@@ -404,6 +405,11 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
+		if (type === "settings") {
+			$("#session-list").text("Loading…");
```

Comment:
Uuuh I feel bad that I missed that but this should be `$("#session-list").html("<p>Loading…</p>");` (or `<p><em>Loading…</em></p>`). I can make the change myself if you give me the go, sorry!